smarts/sstudio/types.py

```diff
@@ -384,7 +384,7 @@ class Flow:
     @property
     def id(self) -> str:
         """The unique id of this flow."""
-        return "flow-{}-{}-".format(
+        return "vehicle-{}-{}-".format(
```

Comment:
Please revert this particular change. No changes to flow should happen.

```suggestion
        return "flow-{}-{}-".format(
```
```suggestion
        return "vehicle-{}-{}-".format(
```